src/main/scala/io/iohk/praos/domain/BlockchainManager.scala

```diff
@@ -0,0 +1,42 @@
+package io.iohk.praos.domain
+
+case class BlockchainManager() {
```

Comment:
Maybe we could replace this with an `object` as it has no configurable parameters?

Reply:
:+1: